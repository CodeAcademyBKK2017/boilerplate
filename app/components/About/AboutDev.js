import React from 'react';
import {
  Text
} from 'react-native';
  
export default class MyNotificationsScreen extends React.Component {
    static navigationOptions = {
      tabBarLabel: 'DEV'
    };
    render () {
      return (
        <Text>DEV</Text>
        // <Button
        //   onPress={() => this.props.navigation.goBack()}
        //   title='Go back home'
        // />
      );
    }
}
