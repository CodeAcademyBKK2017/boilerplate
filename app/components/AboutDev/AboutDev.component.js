import React, {Component} from 'react';
import styles from './AboutDev.style';
import {Text, View} from 'react-native';

export default class AboutDevScreen extends Component {
    static navigationOptions = {
      title: 'About Developer'
    };
      
    render () {
      return (
        
        <View style={styles.container}>
          <Text>About Developer</Text>
        </View>
      );
    }
  
}