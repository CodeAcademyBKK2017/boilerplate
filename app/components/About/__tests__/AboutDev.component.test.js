import 'react-native';
import AboutDev from '../AboutDev.component';
import React from 'react';
import renderer from 'react-test-renderer';

describe('AboutDev', () => {
  it('snapshot test', () => {
    const snapshot = renderer.create(<AboutDev />).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});