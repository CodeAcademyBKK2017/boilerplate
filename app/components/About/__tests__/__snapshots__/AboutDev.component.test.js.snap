// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutDev snapshot test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "flex-start",
      "padding": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    This is About Developer
  </Text>
</View>
`;
